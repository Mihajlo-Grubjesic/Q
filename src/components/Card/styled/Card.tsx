import styled from "styled-components";

export const CardWrapper = styled.div`
  display: flex;
  min-height: 200px;
  background: ${({ theme }) => theme.background};
  border-radius: 4px;
  color: ${({ theme }) => theme.color};
  border: 2px solid ${({ theme }) => theme.color};
`;
