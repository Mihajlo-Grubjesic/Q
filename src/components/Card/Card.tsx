import { CardWrapper } from "./styled/Card";

export const Card = (): JSX.Element => {
  return <CardWrapper>Card</CardWrapper>;
};
