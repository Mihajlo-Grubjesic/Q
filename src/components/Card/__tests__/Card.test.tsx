import { render, screen } from '@testing-library/react';
import { Card } from '../Card';

describe('Card', () => {
  it('renders card', () => {
    const { container } = render(<Card title="title" subtitle="subtitle" />);
    expect(screen.getByText('title')).toBeInTheDocument();
    expect(screen.getByText('subtitle')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
