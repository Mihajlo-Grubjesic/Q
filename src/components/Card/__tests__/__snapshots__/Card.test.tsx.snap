// Vitest Snapshot v1

exports[`Card > renders card 1`] = `
<div>
  <div
    class="sc-dkrFOg joukHU"
  >
    <div
      class="sc-hLBbgP kaMPmA"
    >
      title
    </div>
    <div
      class="sc-eDvSVe cwgtvh"
    >
      subtitle
    </div>
  </div>
</div>
`;
